### ISSUE TITLE
SPFx Teams Webpart not displaying

### ISSUE THREAD

**Issue by vivsunny10** [SharePoint/sp-dev-docs#3004]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

After following the steps to add SPFx webpart in teams tab from following sample link:-https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/using-web-part-as-ms-teams-tab

I am able to add the web part in a sharepoint online modern page . Same web part should be added as tab in microsoft teams .

#### Observed Behavior
I am able to zip the file and upload as a custom app. It is also showing in the apps to add. But while adding the SPfx webpart as a tab its not able to show the properties, its throwing error in console:-

https://presence.teams.microsoft.com/v1/presence/getpresence/
{"message":"The requested resource does not support http method 'GET'."}

NOTE:- I am using a developer tenant to test the feature.

#### Steps to Reproduce
Follow the steps as described in following microsoft sample teams tab web part:-
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/using-web-part-as-ms-teams-tab

**Comment by lokimeyburg** [comment]
Hi @vivsunny10, I'm from the Microsoft Teams... team. That error is pretty unusual and sounds related to our chat presence (aka: online statuses) and not related to the Web Part integration. It's interesting that you're still able to add the app, just not configure the tab. I would be very surprised if the presence service was the cause of your issue. Are there any other error messages? Perhaps in the modal when you configure the tab (or the iFrame _in_ the modal that's hosting your configuration page)? Either way, what would help is if we could take a look at a [Fiddler trace](https://www.telerik.com/fiddler). If you can start the trace right before you select your app to configure as a tab and a second after you don't see anything (like you mentioned). Just to see if there are any other errors or happenings that we should be aware of.

**Comment by vivsunny10** [comment]
Hi @lokimeyburg , I figured out the issue.  Teams where I  was trying to add SPFx web part was created using Microsoft Graph API( currently in preview) . This issue is not reproduced if you directly create the teams from the Out Of box teams interface.

In order to fix the issue, I had to navigate to the "Files" tab of a channel which then creates the required directory. After that when I try to add the SPFx web part it loads without any issue.
